actors/builtin/miner/miner_actor.go

```diff
@@ -1798,7 +1831,7 @@ func handleProvingDeadline(rt Runtime) {
 	// Remove power for new faults, and burn penalties.
 	requestUpdatePower(rt, powerDelta)
 	burnFunds(rt, penaltyTotal)
-	notifyPledgeChanged(rt, big.Sum(newlyVested.Neg(), penaltyTotal.Neg(), pledgeDelta))
+	notifyPledgeChanged(rt, big.Sum(newlyVested.Neg(), penaltyTotal.Neg(), pledgeDelta.Neg()))
```

Comment:
This is breaking the convention of using "delta" to mean a signed quantity that is ready to be added to a total. My ability to understand what's going on is radically reduced.